### PR TITLE
added object keys challenge

### PR DESCRIPTION
**Pull request by benjibuiltit**
first stab at a challenge. feedback welcome.

### CODE REVIEW

File: problems/object-keys/problem.md [hunk 1]
```diff
@@ -1,5 +1,44 @@
----
+Javascript provides a native way of listing all the available keys of an object. This can be helpful for looping through all the properties of an object and manipulating their values accordingly.
```
Comment (ledsun) [inline problems/object-keys/problem.md]: Although it is a minor point, `Javascript` may be difficult for beginners to understand, so please describe `JavaScript` more accurately.

Comment (benjibuiltit) [reply problems/object-keys/problem.md]: I'm not sure that I'm following you here. Are you suggesting I replace `Javascript` with `JavaScript`?

Comment (ledsun) [reply problems/object-keys/problem.md]: Yes 😄

Comment (benjibuiltit) [reply problems/object-keys/problem.md]: Done 👍

**Comment by benjibuiltit** [comment]
What needs to be done to get this merged?

**Comment by ledsun** [comment]
You don't need anything. I was making a decision later.

**Pull request merged**